tests/unit/capi/GEOSIntersectionTest.cpp

```diff
@@ -131,6 +132,36 @@ void object::test<6>
     // No memory leaked
 }
 
+template<>
+template<>
+void object::test<7>
+()
+{
+    geom1_ = GEOSGeomFromWKT("POLYGON ((1 0, 1 1, 0 1, 0 0, 1 0))");
+    geom2_ = GEOSGeomFromWKT("POLYGON ((1 2, 1 3, 0 3, 0 2, 1 2))");
+
+    ensure(geom1_);
+    ensure(geom2_);
+
+    // clear all floating point exceptions
+    feclearexcept (FE_ALL_EXCEPT);
+
+    int old_excepts = fegetexcept(); /* Store old exceptions */
+    feenableexcept(FE_INVALID);
```

Comment:
This is only for debugging, and can be removed in the end, I suppose (but for debugging, it enabled to run GDB on this test and stop at all places that give an issue)